src/test/java/com/rultor/agents/daemons/EndsDaemonITCase.java

```diff
@@ -103,6 +106,37 @@ public void readsExitCodeCorrectly() throws IOException {
     }
 
     /**
+     * EndsDaemon can deprecateDefaultImage.
+     * @throws IOException In case of error.
+     * @todo @todo #754:30min Implement a deprecation message at the start
+     *  of the process if the project is using the default image
+     *  'yegor256/rultor'.
+     */
+    @Test
+    @Ignore
+    public void deprecateDefaultImage() throws IOException {
+        Assume.assumeFalse(SystemUtils.IS_OS_WINDOWS);
+        final Talk talk = new Talk.InFile();
+        final File home = this.start(talk, "");
+        FileUtils.write(new File(home.getAbsolutePath(), "testing"), "12");
+        final Agent agent = new EndsDaemon();
+        agent.execute(talk);
+        final XML xml = talk.read();
+        for (final String path
+            : xml.xpath("/p/entry[@key='merge']/entry[@key='script']")
+        ) {
+            if ("yegor256/rultor".equals(path)) {
+                final String dir = talk.read()
+                    .xpath("/talk/daemon/dir/text()").get(0);
+                MatcherAssert.assertThat(
+                    dir,
+                    StringEndsWith.endsWith("#### Deprecation Notice ####")
```

Comment:
@mkordas  I thought it will be nice for the message to start and end with 
#### Deprecation Notice ####